anyway/models.py

```diff
@@ -871,13 +872,52 @@ class NewsFlash(Base):
     street2_hebrew = Column(Text(), nullable=True)
     non_urban_intersection_hebrew = Column(Text(), nullable=True)
     road_segment_name = Column(Text(), nullable=True)
+    critical = Column(Boolean(), nullable=True)
     newsflash_location_qualification = Column(
         Integer(),
         nullable=False,
         server_default=text(f"{NewsflashLocationQualification.NOT_VERIFIED.value}"),
     )
     location_qualifying_user = Column(BigInteger(), nullable=True)
 
+    def __init__(self, **kwargs):
+        super(NewsFlash, self).__init__(**kwargs)
+        self.critical = self.get_critical()
```

Comment:
Why do you choose to initialize this field in a different way from all other fields?

Reply:
1. this field contains calculation and not the default value that given.
2. only when create a newflash recored
3. the super should init all other filed
4. only when write  a new flash

Reply:
I do not understand why we want to calculate the `critical` field each time we read from the DB. If it needs to be calculated periodically, then we can have a periodic process that updates it. The infographics code can read it from the DB like the other fields.

Reply:
i only want to calc this value when a new newflash is created, there is a way to do this?
i might only calc this when a new newsflash is added, but i am not sure about the location of this in the code

Reply:
I think it is in function `scrape_all` in `anyway/parsers/news_flash.py`

Reply:
i move the method to scrape_extract_store_rss and scrape_extract_store_twitter